src/index.html

```diff
@@ -3,8 +3,8 @@
 
 <head>
   <title>Activity Tracker</title>
-<!--   <link rel="stylesheet" href="normalize.css">
-  <link rel="stylesheet" href="styles.css"> -->
+  <link rel="stylesheet" href="normalize.css">
+  <link rel="stylesheet" href="styles.css">
```

Comment:
Since the sass code is working, we should be able to remove these lines of code permanently.

Reply:
Updated! Good looks in finding this.